### PR TITLE
Ancestry references: note about caret for Windows users

### PR DESCRIPTION
**Pull request by borekb**
Also, the full character names "caret" and "tilde" are mentioned.

Resolves https://github.com/progit/progit2/issues/785

### CODE REVIEW

**Comment by borekb** [comment]
I am not that familiar with AsciiDoc so hopefully the syntax is right.

File: book/07-git-tools/sections/revision-selection.asc [hunk 1]
```diff
@@ -207,6 +207,21 @@ Date:   Thu Dec 11 15:08:43 2008 -0800
     Merge commit 'phedders/rdocs'
 ----
 
+[NOTE]
+.CARET NEEDS TO BE QUOTED ON WINDOWS
```
Comment (ben) [inline book/07-git-tools/sections/revision-selection.asc]: Let's not shout this too loudly. 😄 Change it to `.Escaping the caret on Windows`?

Comment (borekb) [reply book/07-git-tools/sections/revision-selection.asc]: Yes, that's better. I copied the convention from the existing note title ".A SHORT NOTE ABOUT SHA-1" in the document.

**Comment by ben** [comment]
✨ Thanks!

**Pull request merged**